### PR TITLE
fixes deprecation warnings for 0.4

### PR DESCRIPTION
**Pull request by rened**
tests pass locally for 0.3 and 0.4 (tested on osx)

### CODE REVIEW

**Comment by simonster** [comment]
Do you have real tests that you could add there? ATM, the only thing in `test/DICOM_test.jl` is a `using` statement. Even if not, I have no qualms with merging this, as the changes seem pretty innocuous and this is totally broken on 0.4 otherwise.

File: REQUIRE [hunk 1]
```diff
@@ -0,0 +1 @@
+Compat
```
Comment (simonster) [inline REQUIRE]: Should also require `julia 0.3`

Comment (rened) [inline REQUIRE]: Good point, added!

**Comment by rened** [comment]
Added a test (didn't see that there are none...). This started to unearth a few needed updates for 0.4. I fixed what got triggered by the test image, but I assume there are other fixes needed...

**Comment by rened** [comment]
Added a `.travis.yml`, could you please enable it in the travis settings?

**Comment by ihnorton** [comment]
@rened, sorry for missing this. Travis is now enabled, and commits rebased/squashed in 346e20027c867936027264dc6a9f7cf7af30fc9a with `--author=...` as you. Thanks!

(I removed Compat support, but could put it back if you need it...)

**Comment by ihnorton** [comment]
If it would make sense to move this repo somewhere else, let me know and I would be happy to do so. A colleague and I need this and will be working on it, but the repo certainly doesn't have to be under my account.

**Comment by rened** [comment]
@ihnorton no problem at all! ;-) I only need to read dicoms from time to time, and sometimes need to revert to using dcm4che anyway, as some of the dicoms can't be loaded. I tried to look into this but didn't get very far. I am totally fine with keeping the package here!

**Pull request closed without merging**